### PR TITLE
weather app project

### PR DESCRIPTION
**Pull request by wintan1418**
This is a mini web app that gives details about the weather/climatic condition of desired cities, provided it is within the range of its map.
All requirements for this project as been put into practice and it works fine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+DOM Manipulation.
+Images
+Icons
+Linters
+## Getting Started
+Frist Open Your terminal and paste : git clone https://github.com/wintan1418/weather-tracker.
+
+Then cd to the Folder && Open it in your Texteditor.
+
+To Make the website fully working make sure to install the webpack and run the folowing command if yoy don't have it.
```
Comment (GzimAsani) [inline README.md:28]: Please elaborate more and explain it in more detail

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :red_circle: 

## Hello @wintan1418, good job so far I like your project especially how concise you were

### To improve:
- [ ] Your app should have a toggle control to determine if the result will be shown in Celsius or Fahrenheit  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Pathwright](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript/lessons/weather-app))
- [ ] Your app should  change its appearance based on the received data ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from [Pathwright](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript/lessons/weather-app))
- [ ] Be more descriptive in your readme file in `Getting Started` section
- [ OPTIONAL ] In order to look more user friendly add a ‘loading’ component that displays from the time the form is submitted until the information comes back from the API 

### If you have any question with regards to the requirements please add a comment here and I will try to reply ASAP.

# HAPPY CODING :computer: :desktop_computer:

**Comment by wintan1418** [comment]
I did moving of files to another, my environment keeps getting corrupted

**Comment by rukundoeric** [comment]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - Follows gitwork-flow. 💯 
  - Codebase looks clean and professinal. 💯 
  - App is functional and available online. 💯 
  - Interface looks nice. 💯 

- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**